continuous_integration/environment-mindeps-array.yaml

```diff
@@ -8,7 +8,7 @@ dependencies:
   - pyyaml=5.3.1
   - click=7.0
   - cloudpickle=1.1.1
-  - partd=0.3.10
+  - partd=1.1.0
```

Comment:
Though there are some extension types supported by pandas 1.0 (in this case `Int64`), we see failures during shuffling because partd didn't get support for extension arrays until 1.1.0.

Are we okay with bumping the min version of partd? 1.1.0 is still relatively old (~3 years), so I don't imagine this being very impactful, but I could just skip the added tests on our mindeps dataframe build as an alternative.